views/index.html

```diff
@@ -3,49 +3,16 @@
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <link rel="stylesheet" href="../assets/styles/reset.css" />
-    <link rel="stylesheet" href="../assets/styles/main.css" />
-    <link rel="stylesheet" href="../assets/styles/modal.css" />
-    <link rel="stylesheet" href="../assets/styles/tab.css" />
-    <link rel="stylesheet" href="../assets/styles/thumbnail.css" />
-    <script src="./"></script>
     <title>영화 리뷰</title>
+    <link rel="stylesheet" href="./static/styles/index.css" />
   </head>
   <body>
-    <div id="wrap">
-      <header>
-        <div class="background-container" style="background-image: url('${background-container}')">
-          <div class="overlay" aria-hidden="true"></div>
-          <div class="top-rated-container">
-            <h1 class="logo"><img src="../assets/images/logo.png" alt="MovieList" /></h1>
-            <div class="top-rated-movie">
-              <div class="rate">
-                <img src="../assets/images/star_empty.png" class="star" />
-                <span class="rate-value">${bestMovie.rate}</span>
-              </div>
-              <div class="title">${bestMovie.title}</div>
-              <button class="primary detail">자세히 보기</button>
-            </div>
-          </div>
-        </div>
-      </header>
-      <div class="container">
-        <main>
-          <section>
-            <h2>지금 인기 있는 영화</h2>
-            <ul class="thumbnail-list">
-              <!--${MOVIE_ITEMS_PLACEHOLDER}-->
-            </ul>
-          </section>
-        </main>
-      </div>
-
-      <footer class="footer">
-        <p>&copy; 우아한테크코스 All Rights Reserved.</p>
-        <p><img src="../assets/images/woowacourse_logo.png" width="180" /></p>
-      </footer>
-    </div>
-    <!--${MODAL_AREA}-->
+    <div id="root"></div>
+    <footer class="footer">
+      <p>&copy; 우아한테크코스 All Rights Reserved.</p>
+      <p>
+        <img src="../assets/images/woowacourse_logo.png" width="180" alt="" />
```

Comment:
아주 사소하지만 뼈대코드가 변경되면서 여기 경로도 변경되었어요..!
```suggestion
        <img src="./static/images/woowacourse_logo.png" width="180" alt="" />
```
이렇게 하면 이미지도 잘 나옵니다!ㅎㅎ

Reply:
오 수정까먹었네욥 감사합니다~!